specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2024-12-01-preview/managedCassandra.json

```diff
@@ -397,7 +397,7 @@
           "default": {
             "description": "Error response describing why the operation failed.",
             "schema": {
-              "$ref": "cosmos-db.json#/definitions/CloudError"
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
```

Comment:
is this change by design? this cause duplicate schema for your self-defined error response here: https://github.com/Azure/azure-rest-api-specs/blob/ea34a737127701399c649a17747606ecfd2de0f8/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2024-12-01-preview/dataTransferService.json#L541 .